### ISSUE TITLE
Upscale issue #7: Unblur - Strong is way too strong

### ISSUE THREAD

**Issue by minsky91** [Acly/krita-ai-diffusion#1272]
When in Upscale, Refining an image at Scale 1,00x and with Unblur set to Strong results in essentially no visible changes in the upscaled image. Even when the Strength slider is at the highest 50%, the only effect the Refine has under such a strong Unblur is to make the tiles quite visible, in various shades of the source layer color. I tried all styles and samplers under the moon, the same story, no new details or features added to the output except for visible tiles.

Not much sharpness or detail is added to the output when Unblur is set to Medium either, but at least I can see some change in the features rendered, pixel-wise.

I find the three available settings of Unblur (Off, Medium and Strong) a good addition to the functionality but still insufficient for such - often sophisticated - operation as Upscale with Refining, and suggest  enabling full functionality of the Unblur control in the Upscale UI.

**Comment by Acly** [comment]
Is that for SDXL?

The tile/unblur CN lora that's recommended for SD1.5 is not that strong and I think the values work quite well there.

But it might be too high for SDXL. Regardless of how it's presented in the UI, it would still be great to know what a good default is.

**Comment by valery016** [comment]
Yeah, it’s for SDXL. I abandoned 1.5 about half a year ago. In the Krita AI plugin, when Refining I always set custom Unblur values, with the Strength invariably within the range of 0.55-0.60, and the End value fixed at 1.0, to keep the output maximally close to the input and yet allow for sufficient refining by the checkpoint. The plugin’s default variants almost never work for me. Same for Refine in Upscale, but there are some caveats. 

I will post, hopefully, the last report on Upscale issues by tomorrow or the day after, and suggest interface changes that should make this important part of the plugin more efficient and flexible, summarizing thereby all what I have found when testing it.

**Comment by Acly** [comment]
I just went with a slider now, not as noob-friendly, but all the different tile controlnet have different behavior and sometimes there's multiple options for the same base models...

**Comment by minsky91** [comment]
Great, going to check it a.s.ap.

Making it more user-friendly is still needed, especially because - indeed - there can be all kinds of outcomes with different models and samplers for the same refine strength and image guidance value. Still believe a special category of refinement-only presets would be the best solution
